src/gcloud.js

```diff
@@ -0,0 +1,212 @@
+'use strict';
+
+const path = require('path');
+const mkdirp = require('mkdirp-then');
+const Storage = require('@google-cloud/storage');
+const debug = require('debug')('static-storage:gcloud');
+
+
+class GCloud {
+    constructor(configParams) {
+        this.config = configParams;
+        this.tmpFolderPath = configParams.staticstorage.tmpFolderPath.slice(1);
+        this.bucketName = configParams.gcloud.bucketName;
+
+        const credentials = process.env.GOOGLE_APPLICATION_CREDENTIALS;
```

Comment:
15-19 ve arasindaki satirlari ucurabiliriz. yukarida da bahsettigim gibi bunlara gerek yok. kubernetes'in otomatik koymasini saglayabiliyoruz.